recipes/multivisor/meta.yaml

```
@@ -0,0 +1,107 @@
+{% set name = "multivisor" %}
+{% set version = "6.0.1" %}
+
+package:
+  name: multivisor-suite
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/multivisor-{{ version }}.tar.gz
+  sha256: f834f7b6968d478e1e2e24638d4fc29a9a57947fe5642cb266f109745112835b
+
+build:
+  number: 0
+
+outputs:
+  - name: multivisor
+    script: build_core.sh
+    build:
+      noarch: python
+      entry_points:
+        - multivisor=multivisor.server.web:main
+        - multivisor-rpc=multivisor.server.rpc:main
+        - multivisor-cli=multivisor.client.cli:main
+    requirements:
+      host:
+        - pip
+        - python >=3.6
+      run:
+        - gevent >=1.3
+        - python >=3.6
+        - six
+    test:
+      imports:
+        - multivisor
+      commands:
+        - pip check
+      requires:
+        - pip
+
+  - name: multivisor-web
+    build:
+      noarch: python
+    requirements:
+      host:
+        - pip
+        - python >=3.6
+      run:
+        - python >=3.6
+        - {{ pin_subpackage('multivisor', exact=True) }}
+        - flask
+        - werkzeug <2.1
+        - blinker
+        - zerorpc-python
+        - supervisor
+    test:
+      imports:
+        - multivisor.server.web
+      commands:
+        - multivisor --help
+
+  - name: multivisor-rpc
+    build:
+      noarch: python
+    requirements:
+      host:
+        - pip
+        - python >=3.6
+      run:
+        - python >=3.6
+        - {{ pin_subpackage('multivisor', exact=True) }}
+        - zerorpc-python
+        - supervisor
+    test:
+      imports:
+        - multivisor.server.rpc
+      commands:
+        - multivisor-rpc --help
+
+  - name: multivisor-cli
+    build:
+      noarch: python
+    requirements:
+      host:
+        - pip
+        - python >=3.6
+      run:
+        - python >=3.6
+        - {{ pin_subpackage('multivisor', exact=True) }}
+        - maya
+        - requests
+        - prompt_toolkit >=2
+        - blinker
+    test:
+      imports:
+        - multivisor.client.cli
+      commands:
+        - multivisor-cli --help
+
+about:
+  home: https://github.com/tiagocoutinho/multivisor
+  summary: A centralized supervisor UI (web & CLI)
+  license: GPL-3.0-or-later
+  license_file: LICENSE
+
+extra:
```

You might want to add `feedstock-name` here to make sure you get a `multivisor-feedstock` repo and not a `multivisor-suite-feedstock` repo, if that's preferred.

---

```suggestion
extra:
  feedstock-name: multivisor
```

Then I think we need this ^